### ISSUE TITLE
Colors on Windows with bash and Microsoft Visual Code

### ISSUE THREAD

**Issue by ufobat** [perl6/tap-harness6#28]
I am using the Microsoft Visual Code IDE on Windows 10. In Visual Code i've got a terminal which uses the git shell, so it starts a bash. unfortunatally the colors of `prove6` are not displayed correctly. This is what I get:

```
?[32mAll tests successful.
?[0mFiles=5, Tests=46,  115 wallclock secs
Result: PASS
```

The Colors of the prompt or the colors of `ls --color` are displayed correctly.

Not sure if this is TAP::Harness fault but maybe you could provide a command line switch to disable colored output?

**Comment by Leont** [comment]
> The Colors of the prompt or the colors of ls --color are displayed correctly.

What's the value of the `$TERM` environmental variable?

> Not sure if this is TAP::Harness fault but maybe you could provide a command line switch to disable colored output?

Probably it should.

**Comment by ufobat** [comment]
```
$ echo $TERM
cygwin
```

**Comment by Leont** [comment]
I guess that what is needed here is a terminfo/termcap library so we can figure out if the current terminal supports color sequences.

That said, [it should work out of the box](http://www.nivot.org/blog/post/2016/02/04/Windows-10-TH2-%28v1511%29-Console-Host-Enhancements), and I'm not sure why it doesn't

**Comment by Leont** [comment]
This should be fixed in 0.0.7

**Comment by Leont** [comment]
All should work as expected in 0.0.8, and if it doesn't there's now a `--color`/`--no-color` argument (or a `HARNESS_COLOR` environmental variable) to override it to do the right thing.
